### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by HosseinAlipour** [facebook/metro#217]
it's my first time I'm using react native, I tried the 3 steps and failed over and over!
but first step  `watchman: command not found` is given 

```
 error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from                                                                         
 `D:\workspace\react\rn2\node_modules\react-native\Libraries\react-native\react-                                                                         
 native-implementation.js`: Module `AccessibilityInfo` does not exist in the Hast                                                                         
 e module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (D:\workspace\react\rn2\node_modules\metro\src\node-haste\DependencyGraph\ModuleResolution.js:167:1306)
    at ResolutionRequest.resolveDependency (D:\workspace\react\rn2\node_modules\metro\src\node-haste\DependencyGraph\ResolutionRequest.js:80:16)
    at DependencyGraph.resolveDependency (D:\workspace\react\rn2\node_modules\metro\src\node-haste\DependencyGraph.js:237:485)
    at Object.resolve (D:\workspace\react\rn2\node_modules\metro\src\lib\transformHelpers.js:116:25)
    at dependencies.map.result (D:\workspace\react\rn2\node_modules\metro\src\DeltaBundler\traverseDependencies.js:298:29)
    at Array.map (<anonymous>)
    at resolveDependencies (D:\workspace\react\rn2\node_modules\metro\src\DeltaBundler\traverseDependencies.js:294:16)
    at D:\workspace\react\rn2\node_modules\metro\src\DeltaBundler\traverseDependencies.js:159:33
    at Generator.next (<anonymous>)
    at step (D:\workspace\react\rn2\node_modules\metro\src\DeltaBundler\traverseDependencies.js:239:307)
 BUNDLE  [android, dev] ..\..\../index.js ░░░░░░░░░░░░░░░░ 0.0% (0/11), failed.
```

**Comment by Todilo** [comment]
Did you get this fixed? 0.56.1 does not exist and trying to upgrade to rc 0.57 does not seem to work. Some babel conflict. https://github.com/facebook/react-native/issues/20353 appears to be closed but I can't seem to upgrade to 0.57 due to https://github.com/facebook/react-native/issues/20710
